app/views/questions/backtaxes/edit.html.erb

```diff
@@ -9,7 +9,8 @@
       <%=t("views.questions.backtaxes.select_all_that_apply") %>
     </p>
     <div class="form-card__stacked-checkboxes spacing-above-0">
-      <% gyr_backtax_years.reverse.each do |backtax_year| %>
+      <%# gyr_backtax_years.reverse.each do |backtax_year| %>
+      <% [2021, 2022, 2023].each do |backtax_year| %>
```

Comment:
@DrewProebstel, we are confused reading the ticket it seems like we don't want to show 2021 as a backtax option but this PR is adding them no?

Reply:
<img width="631" alt="Screenshot 2025-04-28 at 3 01 13 PM" src="https://github.com/user-attachments/assets/01f8325c-752b-44db-8055-825eaaf21587" />